src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/InboundMeshQueueMultiMessageTest.java

```diff
@@ -94,7 +94,10 @@ private void assertGpOutboundQueueMessages(SoftAssertions softly, Message messag
                 new Customization("identifier.value", IGNORE),
                 new Customization("entry[*].fullUrl", IGNORE),
                 new Customization("entry[*].resource.**.reference", IGNORE),
-                new Customization("entry[*].resource.id", IGNORE)
+                new Customization("entry[*].resource.id", IGNORE),
+                new Customization("entry[*].resource.subject.reference", IGNORE),
+                new Customization("entry[*].resource.id", IGNORE),
+                new Customization("entry[*].resource.performer[*].actor.reference", IGNORE)
```

Comment:
Is it possible to combine any of these using the *?

Reply:
Yea, this change should be already on the main branch. So, if you just merge this branch with main, would be good

Reply:
I must have missed this when merging `main` into this branch. 

Removed these duplicates and combined `Customization` where possible in 3f054a9